_sources/functions_in_cpp/Exercises.rst

```diff
@@ -0,0 +1,314 @@
+Multiple Choice Exercises
+-------------------------
+
```

Comment:
At the beginning of this section (and at the beginning of every end-of-chapter exercise section), let's add this note to students:

`These questions are **completely OPTIONAL**. We provide these exercises to give you more practice with the material that you've learned. Additionally, they may be a helpful resource when reviewing for assessments.`

(Once you've added that note, you can delete the instructions: `Answer the following **Multiple Choice** questions to practice what you have learned in this chapter.`)